cmake/KokkosKernelsConfig.cmake.in

```diff
@@ -14,7 +14,7 @@ SET(Kokkos_ENABLE_OPENMPTARGET @Kokkos_ENABLE_OPENMPTARGET@)
 SET(Kokkos_ENABLE_CUDA         @Kokkos_ENABLE_CUDA@)
 SET(Kokkos_ENABLE_HIP          @Kokkos_ENABLE_HIP@)
 SET(Kokkos_ENABLE_SYCL         @Kokkos_ENABLE_SYCL@)
-SET(Kokkos_ENABLE_PTHREAD      @Kokkos_ENABLE_PTHREAD@) 
+SET(Kokkos_ENABLE_THREADS      @Kokkos_ENABLE_THREADS@) 
```

Comment:
See #1236

Reply:
Based on discussion will drop this change for now

Reply:
#1236 will remove these lines

Reply:
I removed this change with commit 5623619ce6964fd2b9ac22733cbaaaf76e027f6e, force pushed to this branch